src/shared/sleep-config.c

```diff
@@ -125,10 +125,16 @@ int can_sleep_state(char **types) {
 
                 k = strlen(*type);
                 FOREACH_WORD_SEPARATOR(word, l, p, WHITESPACE, state)
```

Comment:
one of these days we should remove the use of FOREACH_WORD from our tree

Reply:
Ack, I'll queue up a patch for that.